sdk/identity/identity/test/manual-integration/AzureFunctions/arm-template.json

```diff
@@ -0,0 +1,236 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
This is great! But a few comments:

1. See [here](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/test-resources.json#L5) where we have a parameter called "baseName" that defaults to resource group name. We should do something similar here - add a baseName, and then use that as the prefix for every resource we create...